chef/cookbooks/ceph/recipes/radosgw.rb

```diff
@@ -68,6 +69,25 @@
   subscribes :restart, "template[/etc/ceph/ceph.conf]"
 end
 
+# Create admin user and add caps to access buckets and users
+uid = "rgw_admin"
+
+Chef::Log.info("Creating RadosGW admin user with capabilities")
+user_add = Mixlib::ShellOut.new("radosgw-admin user create --display-name=RadosGW_Admin --uid=#{uid}")
```

Comment:
Metrics/LineLength: Line is too long. [102/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)